src/ZulipMobile.js

```diff
@@ -16,14 +13,6 @@ export default class ZulipMobile extends Component {
 
   componentWillMount() {
     restore(() => {
-      store.subscribe(() => {
```

Comment:
Yeah but it's in `componentWillMount` which will only happen once. So if you switch accounts it will never get triggered